docs/install.md

```diff
@@ -1,77 +1,197 @@
+    FUSE](https://cloud.google.com/storage/docs/gcs-fuse)
+
+## Core Installation Steps
+
+*   Install python-dev, build essential, pip and setuptools
+    *   `sudo apt-get install python-dev build-essential python-setuptools
+        python-pip`
+*   Install virtualenv and git
+    *   `sudo apt-get install python-virtualenv git`
+*   Create a turbinia user with password-less sudo access **(IMPORTANT)**
+    *   Add your turbinia user to the `disk` group so that you will have
+        permissions to read attached Persistent Disks
+    *   `sudo adduser turbinia disk`
+    *   Add the turbinia user to `/etc/sudoers` by inserting this line:
+        `turbinia ALL=(ALL:ALL) NOPASSWD: ALL`
+*   Log in as turbinia
+    *   `su - turbinia`
+*   Create (once) and activate Virtualenv
+    *   `virtualenv turbinia-env && . turbinia-env/bin/activate`
+    *   ***NOTE:*** the next time you need to use virtualenv, just log in as
+        turbinia and activiate virtualenv without recreating it
+*   Do not exit the Virtualenv until you have completed all the steps!
+*   Continue to [Inside the Virtualenv](#inside-the-virtualenv)
+
+### Inside the Virtualenv
+
+#### Google Cloud SDK, IAM roles, auth credentials
+
+*   Install
+    [google-cloud-sdk](https://cloud.google.com/sdk/docs/quickstart-linux)
+*   Create a [scoped service
+    account](https://cloud.google.com/compute/docs/access/service-accounts)
+    (this is the best option) with the following roles:
+    *   `Cloud Datastore User`: Used by PSQ to store result data, and in the
+        future by the Task Manager to store queriable task data
+    *   `Pub/Sub Editor`: Used by clients to talk to Turbinia, and by the Task
+        Manager to talk to workers
+    *   `Storage Object Admin` and `Storage Legacy Bucket Reader`: Only required
+        on the GCS bucket used by Turbinia, if any. See [GCP Turbinia](#gcp-turbinia)
+        for details
+    *   `Compute Instance Admin`: Used to list instances and to attach disks to
+        instances
+    *   `Service Account User`: Used when attaching disks
+    *   `Cloud Functions Developer`: Used by turbiniactl to query task status
+*   Create a new key for your service account, and then point to it with an
+    environment variable:
+    *   `export
+        GOOGLE_APPLICATION_CREDENTIALS="/home/turbinia/turbinia-service-account-creds.json"`
+*   Add the service account to the gcloud auth **(RECOMMENDED)**
+    *   `gcloud auth list`
+    *   `gcloud auth activate-service-account
+        --key-file=$GOOGLE_APPLICATION_CREDENTIALS`
+*   Alternately you can run Turbinia under your own credentials **(NOT
+    RECOMMENDED)**
+    *   Run `gcloud auth login` (may require you to copy/paste url to browser)
+    *   Or run `gcloud auth application-default login`
+*   Continue to [Build and Configure](#build-and-configure)
+
+#### Build and Configure
+
+*   Install Turbinia
+    *   `sudo apt-get install liblzma-dev`
```

Comment:
Can we also add this package to the previous 'apt-get install' line?

Reply:
done